roles/openshift_metrics/tasks/migrate_cassandra_pvcs.yaml

```diff
@@ -0,0 +1,93 @@
+  block:
+    - name: Get PV name attached to cassandra PVC
+      set_fact:
+        openshift_metrics_pv_name: "{{ openshift_metrics_cassandra_old_pvc['results']['results'][0]['spec']['volumeName'] }}"
+
+    - name: Get PV object attached to cassandra PVC.
+      oc_obj:
+        state: list
+        kind: pv
+        name: "{{ openshift_metrics_pv_name }}"
+        namespace: "{{ openshift_metrics_old_project }}"
+      register: metrics_pv
+
+    - name: Get PV persistent policy attached to cassandra PVC.
+      set_fact:
+        openshift_metrics_cassandra_pvc_persistent_policy: "{{ metrics_pv['results']['results'][0]['spec']['persistentVolumeReclaimPolicy'] }}"
+
+    - name: Set PV reclaim policy to Retain
+      oc_edit:
+        kind: pv
+        name: "{{ openshift_metrics_pv_name }}"
+        separator: '#'
+        content:
+          spec#persistentVolumeReclaimPolicy: "Retain"
+
+    - name:  Generate persistent volume claim templates for namespace {{ openshift_metrics_project }}
+      template:
+        src: pvc.j2
+        dest: "{{ mktemp.stdout }}/templates/hawkular-cassandra-pvc{{ metrics_pvc_index }}.yaml"
+      vars:
+        obj_name: "{{ openshift_metrics_cassandra_pvc_prefix }}-{{ metrics_pvc_index }}"
+        labels:
+          metrics-infra: hawkular-cassandra
+        access_modes: "{{ openshift_metrics_cassandra_pvc_access | list }}"
+        size: "{{ openshift_metrics_cassandra_pvc_size }}"
+        pv_selector: "{{ openshift_metrics_cassandra_pv_selector }}"
+        storage_class_name: "{{ openshift_metrics_cassanda_pvc_storage_class_name | default('', true) }}"
+        volume_name: "{{ openshift_metrics_pv_name }}"
+
+    - slurp:
+        src: "{{ mktemp.stdout }}/templates/hawkular-cassandra-pvc{{ metrics_pvc_index }}.yaml"
+      register: openshift_metrics_cassandra_migration_pvc
+      with_items: "{{ mktemp.stdout }}/templates/hawkular-cassandra-pvc{{ metrics_pvc_index }}.yaml"
+      changed_when: no
+
+    - name: Apply PVC template on namespace {{ openshift_metrics_project }}
+      include_tasks: oc_apply.yaml
+      vars:
+        kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
+        namespace: "{{ openshift_metrics_project }}"
+        file_name: "{{ item.source }}"
+        file_content: "{{ item.content | b64decode | from_yaml }}"
+      with_items: "{{ openshift_metrics_cassandra_migration_pvc.results }}"
+
+    - file:
+        path: "{{ mktemp.stdout }}/templates/hawkular-cassandra-pvc{{ metrics_pvc_index }}.yaml"
+        state: absent
+
+    - name: Wait for Cassandra persistent volume claim to be created on {{ openshift_metrics_project }}
+      oc_obj:
+        state: list
+        kind: pvc
+        name: "{{ openshift_metrics_cassandra_pvc_prefix }}-{{ metrics_pvc_index }}"
+        namespace: "{{openshift_metrics_project}}"
+      register: openshift_metrics_new_pvc
+      until: openshift_metrics_new_pvc.results.stderr is not defined
```

Comment:
should we be checking for a certain field on the object to be available rather than relying on `.stderr` not being defined?